src/challenges.js

```diff
@@ -1,21 +1,43 @@
 /* ======================== CallBacks Practice ============================ */
 const each = (elements, cb) => {
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
+  }
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
 };
 
 const map = (elements, cb) => {
```

Comment:
Good works and is fine.  If you come back to this try and use the `each` function you created above (just a little extra credit/extra thinker if you come back to this file)